content/sensu-core/2.0/reference/checks.md

```diff
@@ -285,7 +285,10 @@ example      | {{< highlight shell >}}"round_robin": false{{< /highlight >}}
 
 |extended_attributes|      |
 -------------|------
-description  | Coming soon.
+description  | Custom attributes to include with the event data, which can be queried like regular attributes.
+required     | false
+type         | JSON object
```

Comment:
The type here should say "Serialized JSON object" - users who are creating checks with `sensuctl create` could be tripped up by that.